### PR TITLE
BUGZ-623: Quiet asan gpu stats

### PR DESCRIPTION
**Pull request by sethalves**
- change a stats variable from int to unsigned long int so it doesn't overflow

https://highfidelity.atlassian.net/secure/RapidBoard.jspa?rapidView=17&modal=detail&selectedIssue=BUGZ-623

### CODE REVIEW

File: libraries/gpu/src/gpu/Context.h [hunk 1]
```diff
@@ -38,7 +38,7 @@ struct ContextStats {
 
     int _RSNumResourceBufferBounded = 0;
     int _RSNumTextureBounded = 0;
-    int _RSAmountTextureMemoryBounded = 0;
+    unsigned long int _RSAmountTextureMemoryBounded = 0;
```
Comment (samcake) [inline libraries/gpu/src/gpu/Context.h]: I would recommend to use the  uint32_t instead here
otherwise, cool

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19874/HighFidelity-Beta-PR15721-2162279.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19874/HighFidelity-Beta-Interface-PR15721-2162279.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19874/HighFidelity-Beta-PR15721-2162279.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19874/HighFidelity-Quest-Beta-PR15721-2162279.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19874/HighFidelity-Beta-PR15721-2162279.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19874/HighFidelity-Beta-Interface-PR15721-2162279.exe

File: libraries/gpu/src/gpu/Context.h (hunk 1 above)
Comment (SamGondelman) [reply libraries/gpu/src/gpu/Context.h]: that only doubles the max value

Comment (sethalves) [reply libraries/gpu/src/gpu/Context.h]: changed to uint64_t

File: libraries/gpu/src/gpu/Context.h [hunk 2]
```diff
@@ -38,7 +38,7 @@ struct ContextStats {
 
     int _RSNumResourceBufferBounded = 0;
     int _RSNumTextureBounded = 0;
-    int _RSAmountTextureMemoryBounded = 0;
+    uint64_t _RSAmountTextureMemoryBounded = 0;
```
Comment (SamGondelman) [inline libraries/gpu/src/gpu/Context.h]: In the place where we add to this, we’re casting the new value to an int, it should be a uint64_t too now.

Comment (sethalves) [reply libraries/gpu/src/gpu/Context.h]: okay, I updated that and the Q_PROPERTY it gets assigned to.

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19899/HighFidelity-Beta-PR15721-eeea8c9.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19899/HighFidelity-Quest-Beta-PR15721-eeea8c9.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19899/HighFidelity-Beta-PR15721-eeea8c9.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19899/HighFidelity-Beta-Interface-PR15721-eeea8c9.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19907/HighFidelity-Beta-PR15721-c5e1f2a.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19907/HighFidelity-Quest-Beta-PR15721-c5e1f2a.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19907/HighFidelity-Beta-PR15721-c5e1f2a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19907/HighFidelity-Beta-Interface-PR15721-c5e1f2a.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19913/HighFidelity-Beta-PR15721-cd1c782.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19913/HighFidelity-Quest-Beta-PR15721-cd1c782.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19913/HighFidelity-Beta-PR15721-cd1c782.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19913/HighFidelity-Beta-Interface-PR15721-cd1c782.exe

**Comment by samcake** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19923/HighFidelity-Beta-PR15721-2fe8979.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19923/HighFidelity-Beta-Interface-PR15721-2fe8979.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19923/HighFidelity-Beta-PR15721-2fe8979.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19923/HighFidelity-Beta-Interface-PR15721-2fe8979.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19923/HighFidelity-Beta-PR15721-2fe8979.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19923/HighFidelity-Quest-Beta-PR15721-2fe8979.apk)

**Pull request merged**